modules/tfm/tfm/boards/common/assert.c

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <pm_config.h>
+#include "autoconf.h"
+#include "region_defs.h"
+
+
+#define IS_ALIGNED_POW2(value, align) (((value) & ((align) - 1)) == 0)
```

Comment:
Would `value % align == 0` work? That way you also lose the requirement that `align` be a power of two